superset/config.py

```diff
@@ -1325,6 +1325,10 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Custom width for screenshots
 ALERT_REPORTS_MIN_CUSTOM_SCREENSHOT_WIDTH = 600
 ALERT_REPORTS_MAX_CUSTOM_SCREENSHOT_WIDTH = 2400
+# Set a minimum interval threshold between executions (for each Alert/Report)
+# Value should be an integer i.e. int(timedelta(minutes=5).total_seconds())
+ALERT_MINIMUM_INTERVAL = 0
+REPORT_MINIMUM_INTERVAL = 0
```

Comment:
Maybe this would be clearer about the expected units of `ALERT_MINIMUM_INTERVAL` and `REPORT_MINIMUM_INTERVAL`.

```suggestion
ALERT_MINIMUM_INTERVAL = int(timedelta(seconds=0).total_seconds())
REPORT_MINIMUM_INTERVAL = int(timedelta(seconds=0).total_seconds())
```

Reply:
ohh that's a great suggestion! I think I'll use `int(timedelta(minutes=0).total_seconds())` since seconds are not really exposed in the report configuration, but it's def better than `0`. 🙌